src/PythonProjectBootstrapper/package/{{ cookiecutter.__empty_dir }}/.github/workflows/standard.yaml

```diff
@@ -232,6 +232,6 @@ jobs:
       release_sources_configuration_filename: .github/release_sources.yaml
     secrets:
       PYPI_TOKEN: {% raw %}${{ secrets.PYPI_TOKEN }}{% endraw %}
-{% if cookiecutter.sign_binaries %}
+{% if cookiecutter.minisign_public_key != 'none' %}

```

Comment:
Nit: same as previous comment

Reply:
Good catch. I will add this to a separate commit after the other changes go in.